src/com/raghav/javabasics/day4/CountIntCharSpecialCharInString.java

```diff
@@ -0,0 +1,60 @@
+package com.raghav.javabasics.day4;
+//Write a program to count number of integers, number of characters and number of special characters.
+public class CountIntCharSpecialCharInString {
+    public static void main(String[] args) {
+        CountIntCharSpecialCharInString result = new CountIntCharSpecialCharInString();
+        result.charCount("abcdefghijklmnopqrstuvwxyz0123456789 !@#$%^&*()_+");
+        result.charCount("");
+        result.charCount(null);
+        result.charCount("cccc0000c ");
+        result.charCount("ascisa%%%-1");
+        result.charCount("abhddabc123*(&");
```

Comment:
Output:
Letter count: 26   Number Count: 10   Special Char Count: 13
Array is Empty/Null
Array is Empty/Null
Letter count: 5   Number Count: 4   Special Char Count: 1
Letter count: 6   Number Count: 1   Special Char Count: 4
Letter count: 8   Number Count: 3   Special Char Count: 3

Reply:
```# of int, char and spe char

Input = “”
for(int i=0;i<=input.length;i++){
	If input[i].isAlpha() {
		letterCount += 1
}
	If input[i].isDigit() {
		digitCount += 1
}
If input[i].isSpecialChar() {
		specialCount += 1
}

}
```